glide.yaml

```diff
@@ -22,13 +28,13 @@ import:
   - package: github.com/technosophos/moniker
     version: ~0.2
   - package: github.com/golang/protobuf
-    version: 1643683e1b54a9e88ad26d98f81400c8c9d9f4f9
+    version: b4deda0973fb4c70b50d226b1af49f3da59f5265
```

Comment:
Is there a reason you updated to protobuf v1.1.0 instead of the more recent v1.2.0?

Also, is there a reason the commit id is used here instead of the tag?